src/pages/RepoPage/shared/constants.ts

```diff
@@ -4,7 +4,7 @@ export const MEASUREMENT_INTERVAL = {
   INTERVAL_1_DAY: 'INTERVAL_1_DAY',
 } as const
 
-export type MEASUREMENT_INTERVAL_TYPE = keyof typeof MEASUREMENT_INTERVAL
+export type MeasurementInterval = keyof typeof MEASUREMENT_INTERVAL
```

Comment:
This was just a lot cleaner looking to me, changed all the other instances to use this as well

Reply:
Nice. Not sure what pattern we go for elsewhere but looks like at least in this file we sometimes have the enum options have a pluralized name to help distinguish it

Reply:
Yeah, this is something we'll need to hash out as a team and make a norm around. I see all sorts of patterns throughout the codebase but I think having a shared consts file or directory with labeled files for each const makes the most sense to me